src/cli/README.md

```diff
@@ -561,6 +562,34 @@ Set the Thread Domain Name for Thread 1.2 device.
 Done
 ```
 
+### dua addr
+
+Get the current Thread Domain Unicast Address manually specified for Thread 1.2 device.
+
+```bash
+> dua addr
```

Comment:
Considered using single `dua` to show/configure the manual DUA as the command name `dua` itself could represent `address`. 
```
> dua
fd00:7d03:7d03:7d04:0:0:2:1
Done
> dua addr fd00:7d03:7d03:7d04::2:1
Done
```
However since we need `dua clear` to clear the manual setting, to keep sub-command style, so introduce the sub command `addr` here.

As to the sub-command, `addr` v.s. `fixed`, I would prefer `addr` a bit as it is more descriptive in representing an address. (and another reason, not sure but it might be useful in the future, we may easily introduce one sub-command `iid` so that regardless of the domain prefix, the manual `iid` configuration could always take effect).

I would prefer to keep it as is or change it to the style inlined in this comment above.

Reply:
Users could easily think that `dua addr` should print the current DUA address assigned to Thread network interface? 

Besides, I would suggest putting all manual (fixed) DUA operations under one sub command like `dua fixed` or `dua addr`, which ever is chosen eventually.

Reply:
Users SHOULD read `src/cli/README.md` to learn what the command is used for and how to use it properly.